controllers/istio_controller.go

```diff
@@ -160,11 +160,13 @@ func (r *IstioReconciler) Reconcile(ctx context.Context, req ctrl.Request) (ctrl
 		return ctrl.Result{}, err
 	}
 
-	logger.Info("Installing components", "values", values)
-	err = r.installHelmCharts(ctx, istio, values)
+	mergedValues := mergeOverwrite(istio.Spec.GetRawValues(), values)
```

Comment:
You need to rebase, but when you do, please make sure to use the existing `values` variable instead of introducing a new variable called `mergedValues`. Since we already make several other changes to the initial `values`, there's no need to use a new variable when we merge in the `rawValues` (this new variable might even cause bugs down the line if someone uses `values` instead of `mergedValues`).